README.md

```diff
@@ -1,82 +1,24 @@
 # Trin
-Trin is an Ethereum "portal": a json-rpc server with nearly instant sync, and
-low CPU & storage usage.
 
-Trin does this by making these tradeoffs:
-- Trusts miners to include valid state transitions, to skip sync
-- Shards state across a (new) p2p network, to reduce local storage needs
+Trin is a Rust implementation of a [Portal Network](https://github.com/ethereum/portal-network-specs) client.
 
-This should sound similar to a light client. It is, but with a peer-to-peer
-philosophy rather than the LES client/server model, which has introduced
-challenges in an altruistic environment.
-
-## Repository Structure
-
-The portal protocol is a collection of networks. Trin needs to connect to all of them.
-
-The repository is structured as follows:
-
-- `trin/`: The main entry point to run a holistic Ethereum portal client
-- `trin-history/`: The chain history network
-- `trin-state/`: The state network
-
-## Ready for production?
-
-LOL, not even a little bit. At the last readme update, this was simply a proxy
-to Infura for all inbound requests, and doesn't validate any answers against
-state roots.
-
-Trin will proxy at least *some* requests to Infura for quite a while, but the
-plan is to incrementally reduce the reliance on Infura, as more trin
-functionality becomes available.
+The Portal Network is still in the research phase, so understand that this client is experimental.
 
 ## How to use Trin
 
-Checkout out the [Getting Started](/docs/getting_started.md) guide to quickly get up and running with Trin.
+Check out the [Getting Started](/docs/getting_started.md) guide to quickly get up and running with Trin.
 
-## CLI Options
-```sh
-trin 0.0.1
-carver
-Run an eth portal client
+## Experimental Status
 
-USAGE:
-    trin [OPTIONS]
+Trin is a prototype Portal Network client. This implementation and the Portal Network specificiation will continue to co-evolve.
 
-FLAGS:
-        --enable-metrics    Enable prometheus metrics reporting (requires --metrics-url)
-    -h, --help              Prints help information
-        --internal-ip       (For testing purposes) Use local ip address rather than external via STUN.
-    -V, --version           Prints version information
+In this stage of development, Trin relies on Infura to respond to some requests that it cannot fulfill itself. Additionally, Trin lacks comprehensive data validation.
```

Comment:
Hmmm, technically, we only use Infura for a single endpoint. `eth_blockNumber`. This is something I've been thinking about recently, because if we advertise ourselves as a "full" client that proxies all unsupported endpoints to infura, then we need to implement that.

Reply:
Yeah, that's why I went with the broad "some" descriptor. The point I was trying to get across here is that trin may rely on centralized infrastructure depending on the endpoint.